Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Feed.kt

```diff
@@ -47,55 +46,62 @@ import com.example.jetsnack.ui.components.JetsnackSurface
 import com.example.jetsnack.ui.components.SnackCollection
 import com.example.jetsnack.ui.theme.AlphaNearOpaque
 import com.example.jetsnack.ui.theme.JetsnackTheme
-import com.example.jetsnack.ui.utils.navigationBarsPadding
+import com.example.jetsnack.ui.utils.statusBarsHeight
 import com.example.jetsnack.ui.utils.statusBarsPadding
 
 @Composable
 fun Feed(
     onSnackClick: (Long) -> Unit,
     modifier: Modifier = Modifier
 ) {
-    val data = remember { SnackRepo.getSnacks() }
     val filters = remember { SnackRepo.getFilters() }
+    val snackCollections = remember { SnackRepo.getSnacks() }
+    Feed(
+        filters,
+        snackCollections,
+        onSnackClick,
+        modifier
+    )
+}
+
+@Composable
+private fun Feed(
+    filters: List<Filter>,
+    snackCollections: List<SnackCollection>,
```

Comment:
Nit: Feels like passing `snackCollections` as an argument first and then `filters` reads better when calling this function.

Reply:
+1

Reply:
Yeah, agreed.